pkg/operator/ceph/file/filesystem.go

```diff
@@ -193,10 +216,10 @@ func (f *Filesystem) doFilesystemCreate(context *clusterd.Context, cephVersion c
 		}
 	}
 
-	var dataPoolNames []string
+	var dataPoolsName []string
+	dataPoolNames := generateDataPoolNames(f, spec)
```

Comment:
A whole new variable `dataPoolNames` is created here. So now we have two variables `dataPoolNames` and 'dataPoolsName`.  And `dataPoolsName` is always empty.  This looks like the reason for integration tests failure.

Reply:
Done, thanks